lib/trellis/vagrant.rb

```diff
@@ -79,6 +79,22 @@ def post_up_message
   msg << "\n* You can SSH into the machine with `vagrant ssh`."
   msg << "\n* Then navigate to your WordPress sites at `/srv/www`"
   msg << "\n  or to your Trellis files at `#{ANSIBLE_PATH_ON_VM}`."
+  sites = YAML.load_file(File.join(ANSIBLE_PATH, 'group_vars/development/wordpress_sites.yml'))
+  wordpress_sites = sites['wordpress_sites']
```

Comment:
This is already available as `trellis_config.wordpress_sites`

Reply:
trellis_config wasn't available so I loaded it the same way the Vagrantfile does.

Reply:
Did you actually get an error? This method is only used in the `Vagrantfile` which has `trellis_config` in scope.

If for some reason it doesn't work, you should change the `post_up_message` method to accept an `trellis_config` argument so it can be passed in.